test/unitTests.js

```diff
@@ -39,35 +38,40 @@ describe('Incorrect email and correct password log in test', function() {
            })
        });
 
- it('Log in should return null with the wrong email and password', async function() {
-       const response = await AuthenticationController.login(incorrectLogInEmail,incorrectLogInPassword)
-       assert.equal(response,null);
-        })
-    });
-describe('Incorrect email and correct password log in test', function() {
- it('Log in should return null with the incorrect email and correct password', async function() {
-       const response = await AuthenticationController.login(incorrectLogInEmail,correctLogInPassword)
-       assert.equal(response,null);
-        })
-    });
-
 
 // both tests with correct emails should return a not null user, even if the password is incorrect
 // the password is checked in the AuthenticationService.js file
 describe('Correct email and incorrect password log in test', function() {
- it('Log in should return null with the correct email and incorrect password', async function() {
+ it('Log in should not return null with the correct email and incorrect password', async function() {
        const response = await AuthenticationController.login(correctLogInEmail,incorrectLogInPassword)
```

Comment:
Testing all of these edge cases within our log in unit tests definitely allows us to practice better coding practices and ensures that any additional changes to our code base can be verified through the unit tests before merging to our master branch! Great job